src/errors.ts

```diff
@@ -31,24 +31,26 @@ export const APIErrorCodes: Record<string, { name: string; retryable: boolean }>
   '99': { name: 'AppSuspendedError', retryable: false },
 };
 
-type APIError = Error & { code: number; isWSFailure?: boolean };
+type APIError = Error & { code: number; isWSFailure?: boolean; StatusCode?: number };
 
-export function isAPIError(error: Error): error is APIError {
-  return (error as APIError).code !== undefined;
+export function isAPIError(error: unknown): error is APIError {
+  return (error as APIError)?.code !== undefined;
 }
 
-export function isErrorRetryable(error: APIError) {
+export function isErrorRetryable(error: any) {
   if (!error.code) return false;
   const err = APIErrorCodes[`${error.code}`];
   if (!err) return false;
   return err.retryable;
 }
 
-export function isConnectionIDError(error: APIError) {
-  return error.code === 46; // ConnectionIDNotFoundError
+export function isConnectionIDError(error: unknown) {
+  //
+
+  return (error as any)?.code === 46; // ConnectionIDNotFoundError
```

Comment:
I know nothing actually changed here, but could we perhaps make this a `const`?  The 46, that is.

Reply:
Yeah, I see the point, but I don't know if its worth doing so. Thanks for the feedback, much appreciated 🙏